### PR TITLE
Add bin script for deployment

### PR DESCRIPTION
**Pull request by matthieuRioual**
- Related to https://github.com/jhipster/jhipster-lite/issues/1132

### CODE REVIEW

File: LICENSE.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
 
-    Copyright 2013-2022 the original author or authors from the JHipster project
+    Copyright 2018-2021 the original author or authors from the JHipster project
```
Comment (pascalgrimaud) [inline LICENSE.txt]: plz revert the change on LICENSE

File: NOTICE [hunk 2]
```diff
@@ -1,4 +1,4 @@
 JHipster
-Copyright 2013-2022 the original author or authors from the JHipster project.
+Copyright 2018-2021 the original author or authors from the JHipster project.
```
Comment (pascalgrimaud) [inline NOTICE]: same, you need to revert

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]
maybe missing `buildpack.toml` at root folder?

File: LICENSE.txt (hunk 1 above)
Comment (matthieuRioual) [reply LICENSE.txt]: Oh yes sorry, i thought that was a typo or a copy/paste error, i will revert

**Review by pascalgrimaud — APPROVED** [review]
